locales/de/calendar.json

```diff
@@ -0,0 +1,48 @@
+{
+  "date": "Datum",
+  "time": "Uhrzeit",
+  "event": "Formel 1 Grand Prix Veranstaltungen",
+  "notice": {
+    "text": "Die ersten acht Rennen des revidierten Kalenders unten. Times are TBC along with additional races.",
```

Comment:
Could be 'Die ersten 8 Rennen des neuen Kalenders sind unten aufgelistet. Die Startzeiten und weitere Rennen müssen erst noch bestätigt werden.'